sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/MarshalSpec.kt

```diff
@@ -15,12 +15,7 @@
  */
 package com.squareup.sqldelight
 
-import com.squareup.javapoet.ClassName
-import com.squareup.javapoet.FieldSpec
-import com.squareup.javapoet.MethodSpec
-import com.squareup.javapoet.ParameterizedTypeName
-import com.squareup.javapoet.TypeSpec
-import com.squareup.javapoet.TypeVariableName
+import com.squareup.javapoet.*
```

Comment:
also specifically for star imports I was running into issues with java-code-styles for whatever reason so if you also have that do

Android Studio -> Preferences -> Editor -> Code Style -> Kotlin -> Imports -> Check the boxes that say "Use single name imports"